frontend/src/hooks/useGlobalLocalStorage.ts

```diff
@@ -2,7 +2,7 @@ import { createLocalStorage } from 'utils/createLocalStorage';
 
 interface IGlobalStore {
     favorites?: boolean;
-    hiddenEnvironments?: Set<string>;
+    hiddenEnvironments?: Array<string>;
```

Comment:
this type was incorrect before since we can't persist sets in local storage, only arrays